css/style.css

```diff
@@ -0,0 +1,209 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font: 12px/18px "Lucida Grande", "Lucida Sans Unicode", Helvetica, Arial, Verdana, sans-serif;
+  background-color: #eee;
+  color: #333;
+  margin: 0;
+  background-image: url("../images/bg-img.jpg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: 0% 180%;
+}
+
+.header-container {
+  background-color: transparent;
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  height: 550px;
+}
+
+.nav-container {
+  height: 35px;
+  width: 70%;
+  border-radius: 5px;
+  margin: 0 auto;
+  background: rgb(128, 128, 128);
+  background: linear-gradient(rgb(128, 128, 128) 48%, rgb(114, 112, 112) 50%);
+  display: flex;
+  margin-top: 10%;
+}
+
+.logo {
+  background-image: url("../images/logo.png");
+  background-position: 50% 0;
+  background-repeat: no-repeat;
+  margin: 3px auto;
+  cursor: pointer;
+}
+
+.nav-item {
+  padding: 6px;
+  width: 100px;
+  margin: 0;
+  text-align: center;
+  border-right: 1px solid rgb(80, 77, 77);
+}
+
+.nav-item:hover {
+  background-color: rgb(128, 128, 128);
+}
+
+#form-item {
+  width: 155px;
+  border: none;
+  height: 34px;
+  padding: 0;
+  background-color: transparent;
+}
+
+.nav-item a {
+  color: #fff;
+  text-decoration: none;
+  width: 100%;
+  font: normal normal normal 12px/18px "Lucida Grande", "Lucida Sans Unicode", Helvetica, Arial, Verdana, sans-serif;
+}
+
+input[type="text"] {
+  width: 130px;
+  border-radius: 15px;
+  height: 25px;
+  padding: 0;
+  background-color: transparent;
+  border-color: none;
```

Comment:
`border-color` doesn't accept `none` as its value. You can use `transparent` as a workaround if you don't want colors.